src/components/m-datepicker/_template.js

```diff
@@ -73,26 +74,28 @@ export default ({
   const specificYears = getSpecificYears({ lowerEndYear, higherEndYear });
   return html`
     <article class="${classes} m-datepicker__article">
-      <axa-dropdown data-month="true" class="m-datepicker__dropdown m-datepicker__dropdown__month js-datepicker__dropdown__month"
-        size="sm" value="${startDate.month}"
-        items="${JSON.stringify(getAllLocaleMonthsArray(locale).map((month, index) => ({
-          name: month, url: '#', value: index,
-        })))}">
-      </axa-dropdown>
-      <axa-dropdown data-year="true" class="m-datepicker__dropdown m-datepicker__dropdown__year js-datepicker__dropdown__year"
-        size="sm" value="${startDate.year}"
-        items="${JSON.stringify(specificYears.map(year => ({
-          name: year, url: '#', value: year,
-        })))}">
-      </axa-dropdown>
+    <axa-dropdown data-month="true" class="m-datepicker__dropdown m-datepicker__dropdown__month js-datepicker__dropdown__month"
```

Comment:
indentation is broken now, 2 less spaces at each line.

I think we should also spread `class` attributes on separate lines...

Reply:
I removed all datepicker changes. they were not meant to be in this PR.